src/modules/visualBuilder/pageBuilder.js

```diff
@@ -0,0 +1,146 @@
+export const initialiseCTBuilder = (url, variant, details) => {
```

Comment:
general review, lets add jsdoc comments to all functions
https://jsdoc.app/

Reply:
This is Done